atm.js

```diff
@@ -1,21 +1,95 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Checking account - deposit function
+  $(".depositButton").on("click",function(){
+    var deposit  = parseInt($(".depositInput").val())
+    var balance =$(".balance.checking").text()
+    balance = parseInt(balance.replace("$",""))
+    var newBalance = deposit + balance
+    $(".balance.checking").text("$"+ newBalance)
+    classChangeChecking(newBalance);
+  })
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// Checking account - withdraw function
+  $(".depositWithdraw").on("click",function(){
+    var deposit  = parseInt($(".depositInput").val())
+    var balance =$(".balance.checking").text()
+    balance = parseInt(balance.replace("$",""))
 
-}
+    var savingsGlobalBalance =$(".balance.saving").text()
+    savingsGlobalBalance = parseInt(savingsGlobalBalance.replace("$",""))
 
-function checkingWithdrawal(amount){
+    if(deposit <= balance)
+    {
+      var newBalance = balance - deposit
+    }
+    else if(deposit <= (balance + savingsGlobalBalance))
+    {
+      newBalance= 0;
+      var overdraftBalance = savingsGlobalBalance + balance - deposit
+      $(".balance.saving").text("$"+ overdraftBalance)
+    }
+    else
+    {
+      newBalance=balance
+    }
+    $(".balance.checking").text("$"+ newBalance)
+    classChangeChecking(newBalance);
+  })
 
-}
+//Saving account - deposit function
+  $(".savingsDepositButton").on("click",function(){
+    var deposit  = parseInt($(".savingsInput").val())
+    var balance =$(".balance.saving").text()
+    balance = parseInt(balance.replace("$",""))
+    var newBalance = deposit + balance
+    $(".balance.saving").text("$"+ newBalance)
+    savingsGlobalBalance= newBalance;
 
-function savingsDeposit(amount){
+    classChangeSaving(newBalance)
```

Comment:
I think you can remove the `savingsDeposit` function and just call `classChangeSaving` here.. no?